test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -125,11 +157,7 @@ def hook(context):
     global previousErrorResultCount
```

Comment:
why use the global variable twice? can't we use once

Reply:
no twice!
functions have their own scopes